docs/deploy-configure.md

```diff
@@ -149,22 +149,21 @@ The value of the `spec.arbiters` field must be:
 
 **NOTE**: At least one replica set member must not be an arbiter.
 
-Consider the following MongoDBCommunity resource definition example:
+Consider the following MongoDBCommunity resource definition example, with a PSS
+(Primary-Secondary-Secondary) configuration:
 
 ```yaml
 apiVersion: mongodbcommunity.mongodb.com/v1
 kind: MongoDBCommunity
 metadata:
   name: example-mongodb
 spec:
-  members: 3
   type: ReplicaSet
+  members: 3
   version: "4.2.7"
 ```
 
-To add arbiters:
-
-1. Edit the resource definition.
+To add arbiters, edit the resource definition.
```

Comment:
consider reverting this change. I like that your construction is simpler, but removing the numbered list item here and not at https://github.com/mongodb/mongodb-kubernetes-operator/pull/970/files#diff-f78009da6728161bca1880b44e8c39f4d02497f945c161e525ae024a68935c11R182 leaves only a single numbered list item here that starts with `2`.
```suggestion
To add arbiters: 

1. Edit the resource definition.
```